rules-java/src/main/java/org/jboss/windup/rules/apps/java/ip/DiscoverStaticIPAddressRuleProvider.java

```diff
@@ -14,32 +14,46 @@
 import org.ocpsoft.rewrite.context.EvaluationContext;
 
 /**
- * Finds files that contain potential static IP addresses, determined by regular
- * expression.
+ * Finds files that contain potential static IP addresses, determined by regular expression.
  * 
  * @author <a href="mailto:bradsdavis@gmail.com">Brad Davis</a>
  */
-public class DiscoverStaticIPAddressRuleProvider extends WindupRuleProvider {
-	@Override
-	public Class<? extends RulePhase> getPhase() {
-		return MigrationRules.class;
-	}
+public class DiscoverStaticIPAddressRuleProvider extends WindupRuleProvider
+{
+    @Override
+    public Class<? extends RulePhase> getPhase()
+    {
+        return MigrationRules.class;
+    }
 
-	@Override
-	public Configuration getConfiguration(GraphContext context) {
+    @Override
+    public Configuration getConfiguration(GraphContext context)
+    {
+        return ConfigurationBuilder
+                    .begin()
+                    .addRule()
+                    // for all files ending in java, properties, and xml,
```

Comment:
I added them because i figured we would use it as an example.  Getting Lincoln to walk me through creating a rule made it straight forward.  I am sure other people wish they had a Lincoln living in the same city.  But, you can always just put it in a how to.  ;)